csunplugged/topics/content/en/image-representation/unit-plan/lessons/colour-by-numbers.md

```diff
@@ -0,0 +1,157 @@
+# Colour by numbers
+
+
```

Comment:
Lots of double newlines in this document which should be changed to a single newline separator.